src/platform/plugins/shared/telemetry/schema/oss_platform.json

```diff
@@ -9927,6 +9927,38 @@
         }
       }
     },
+    "favorites": {
```

Comment:
@elastic/kibana-core If this is added here, then should this be also deleted from somewhere? 🤔

Reply:
There is a bug in the `telemetry_check` script that didn't let it update completely removed properties.

I opened #205613 to resolve this. Please wait until it is merged and then merge upstream into this PR. Otherwise, there is a chance of getting a broken main.

Reply:
Awesome, thanks! will wait for that PR

Reply:
@dokmic, looks like nothing has changed after the merge. Should I remove it manually?